product_docs/docs/pgd/5/rel_notes/pgd_5.4.0_rel_notes.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: "EDB Postgres Distributed 5.4.0 release notes"
+navTitle: "Version 5.4.0"
+---
+
+Released: xx xxx 2024
+
+EDB Postgres Distributed version 5.4.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.4. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+## Highlights of EDB Postgres Distributed 5.4.0
+
+* Group Commit and Parallel Apply are now compatible with each other.
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
+
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres Advanced Server or EDB Postgres Extended 
+upgrades with an upgrade of EDB Postgres Distributed.
+!!!
+
+## Features
+
+| Component | Version | Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| PGD | 5.4.0 | Group Commit and Parallel Apply can work together with a dedicated writer for Group Commit. | | /* TBC (To Be Committed) */
+
+## Enhancements
+
+| Component | Version  |Description | Addresses |
+|-----------|---------|-------------|-----------|
+| BDR       | 5.4.0   | Reduced WAL file retention when decoding worker enabled by using more compact LCR files. | | /* 00-wal-reduction-with-LCR */
```

Comment:
This is not in 5.4 and the 00-wal-reduction-with-LCR files is not in REL_5_STABLE branch, don't use master for these please.